tutorials/scripting/c_sharp/c_sharp_global_classes.rst

```diff
@@ -21,6 +26,35 @@ The ``MyNode`` type will be registered as a global class with the same name as t
 
 .. image:: img/globalclasses_addnode.webp
 
+The *Select a Node* window for the ``MyNode`` exported property filters the list
+of nodes in the scene to match the assignment restriction.
+
+.. code-block:: csharp
+
+    public partial class Main : Node
+    {
+        [Export]
+        public MyNode MyNode { get; set; }
+    }
+
+
+.. image:: img/globalclasses_exportednode.webp
+
+The inheritance hierarchy is used if a custom type isn't registered.
+
+.. code-block:: csharp
+
+    public partial class MySpecializedNode : MyNode
+    {
+    }
+
+Inspector assignments to an export of type ``MySpecializedNode`` are restricted
+to ``MyNode`` and derivative types. This is because ``MySpecializedNode`` isn't
+registered, but its parent, ``MyNode``, is.
```

Comment:
I tried this again, and it doesn't behave like I wrote it here. It only restricts to `Node`. 😕 I'm not sure if the behavior actually changed, if I just read my results wrong on my first attempt, or if something's subtly different in this attempt.

It seems the most specific built-in node type is used in this case, too.